website/src/react-native/css/react-native.css

```diff
@@ -1545,3 +1545,53 @@ input#algolia-doc-search:focus {
   background-color: hsl(198, 100%, 96%);
   color: #3B3738;
 }
+
+.params, .props
+{
+    border-spacing: 0;
+    border: 0;
+    border-collapse: collapse;
+}
+
+.params .name, .props .name, .name code {
+    color: #4D4E53;
+}
+
+.params td, .params th, .props td, .props th
```

Comment:
Good old CSS specificity :)
At some point this file will become beyond maintainability threshold.
Soon.